convene-web/README.md

```diff
@@ -18,17 +18,17 @@ First, ensure your development environment has:
 1. Node (See [.nvmrc](./.nvmrc) for version)
 1. [PostgreSQL 12]. (Note: For people using [Docker], a [docker-compose.yml]
    file has been included for convenience.)
-1. Copy `[.env.example]` to `.env` and make any changes: `cp .env.example
-   .env`
+1. Copy `convene-web/.env.example` to `convene-web/.env` and make any changes: `cp convene-web/.env.example
+   convene-web/.env`.
```

Comment:
@user512 I noticed that there is also a different `.env.example` file in the root of the repo. Not sure if that should also be mentioned here in the README. Please let me know. 🙏

Reply:
The `.env.example` isn't related to `convene-web` so we can leave that.